git_machete/cmd.py

```diff
@@ -3861,9 +3936,9 @@ def check_required_param(in_args: List[str], allowed_values_string: str) -> str:
             param = check_optional_param(parse_options(args))
             # No need to read definition file.
             usage(param)
-        elif cmd == "hub":
-            hub_allowed_values = "anno-prs|create-pr|retarget-pr"
-            param = check_required_param(parse_options(args, "", ["draft"]), hub_allowed_values)
+        elif cmd == "gh":
```

Comment:
Judging by how `gh`'s CLI looks like (https://cli.github.com/manual/index), I'd rather expect a slightly different set of subcommands (sth like `pr create`, `pr anno`, `pr retarget`)... although that's to be discussed.

`git machete gh pr create` would follow the conventions of the original `gh`, but at the same time require an extra rather redundant argument to the command line... maybe `git machete gh-pr create` would be a good compromise? 🤔

Reply:
Also, maybe actually it'd be most reasonable to ditch following either `hub` or `gh` CLI completely, and name our command like `git machete github` to avoid any association with either existing CLI 🤔